CHANGES.rst

```diff
@@ -2,6 +2,7 @@ Version 3.1.2
 -------------
 
 - Fix issue with calling ``repr()`` on ``SQLAlchemy`` instance with no default engine. :issue:`1295`
+-  No longer call ``datetime.utcnow()`` in the test suite. :issue:`1303`
```

Comment:
Typically we don't include a changes entry for tests and docs changes, as they don't affect how the user uses the library. Sorry, missed this earlier.